fe/fe-core/src/main/java/com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java

```diff
@@ -115,6 +115,11 @@ public Void visitCreateMaterializedViewStatement(CreateMaterializedViewStatement
                             "Materialized view only supports olap table, but the type of table: " +
                                     table.getName() + " is: " + table.getType().name());
                 }
+                if (table instanceof MaterializedView) {
+                    throw new SemanticException(
+                            "Materialized view not supports base on Materialized view so far, but the type of table: " +
```

Comment:
creating a materialized view from materialized view is not supported now. The type of table: + table.getName is a materialized view.